src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
The layout and Zeplin design don't match. Have a look at [this document](https://www.notion.so/Pixel-perfect-layouts-5b37e3c869cf45a4bf05736d94d75394) for more info

![image](https://user-images.githubusercontent.com/9722792/77822342-fea6e080-70f1-11ea-876f-2afc09aed820.png)